### ISSUE TITLE
Namespace issue after installation

### ISSUE THREAD

**Issue by r-christie** [sabresaurus/SabreCSG#232]
Hi, I'm using Unity 2018.4, I'm trying out Sabre since the features look really good, and simple to use. 

However after copying the SabreCSG-master folder into the assets folder then opening my project, it imports the Sabre Files, as per the instructions, I then get the error below. 

Assets\SabreCSG-master\Scripts\CSGModel.cs(1783,36): error CS0234: The type or namespace name 'FindObjectsOfTypeAll' does not exist in the namespace 'Resources' (are you missing an assembly reference?)

**Comment by r-christie** [comment]
Fixed locally by editing the classes: CSGModel,

Every use of the namespace Resources was set to use UnityEngine.Resources explicitly, since I had another library (Panda AI) which also has a Resources namespace which conflicted.

A similar issue exists for the CSGModelBase class.

**Comment by sabresaurus** [comment]
Unless I'm misunderstanding, it's surprising that a library would define a namespace as a root level "Resources", the official Microsoft [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces) suggest prefixing with a organisation name and to avoid generic names

Do you have a link to the Panda AI plugin? I had a google and looked at [Panda BT Free](https://assetstore.unity.com/packages/tools/ai/panda-bt-free-33057) but couldn't see a Resources namespace

**Comment by r-christie** [comment]
Hmm you have the right library there, maybe I added the resources folder.
It's a long time ago I added the library and started using it, and I was
new to Unity at the time. I think I remember needing to put the AI scripts
and the classes they used in the same folder, and that folder was inside
the resources folder I created. They were given the namespaces based on the
folder structure of course which caused the issue.

This is probably a rare enough clash you won't need to do anything for
other developers, it's a bug on my side.

Thanks for looking into it anyways, it helped me fix it.


Russell
